addons/account/tests/test_invoice_taxes.py

```diff
@@ -448,70 +448,46 @@ def test_tax_calculation_foreign_currency_large_quantity(self):
         ------------------
         2.82       | 20000     | 21% not incl
         '''
-        self.env['res.currency.rate'].search([]).unlink()
-        self.currency_usd_id = self.env.ref("base.USD")
-        self.currency_chf_id = self.env.ref("base.CHF")
-        # make sure that we have USD as base...
-        self.cr.execute("UPDATE res_company SET currency_id = %s WHERE id = %s", 
-            [self.currency_usd_id.id, self.env.company.id])
         self.env['res.currency.rate'].create({
-            'currency_id': self.currency_chf_id.id,
-            'rate': 1.1726,
-            'name': '2001-01-01'})
+            'name': '2018-01-01',
+            'rate': 0.654065014,
```

Comment:
Why are you changing the rate? The point of rounding issues is to keep the specific values triggering the problem, not using other values.

Reply:
Because the rate in main_company is 1.5287 for USD (demo data) but 1 in the current setup. The rate doesn't matter in fact but I'm sure this one is also representing the current issue.

Reply:
Edit: I probably copy-pasted the rate for this one. I'll restore the previous rate